src/PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.cpp

```diff
@@ -300,24 +343,24 @@ double Enthalpy<LowDensityEoS>::evaluate_coefficients(
          const std::vector<double>& basis_coefficients, const double initial) {
         double sum = initial;
         for (size_t index = 0; index < basis_coefficients.size(); index++) {
-          sum += basis_coefficients[index] *
-                 basis_function(evaluation_point, index);
+          sum = std::fma(basis_coefficients[index],
+                         basis_function(evaluation_point, index), sum);
         }
         return sum;
       };
-  const double k = coefficients.trig_scale;
+  const double k_times_x = coefficients.trig_scale * x;
   const double polynomial_contribution =
       evaluate_polynomial(coefficients.polynomial_coefficients, x);
   const double sin_contribution = evaluate_with_function_basis(
-      [k](double z, int n) { return sin((n + 1) * k * z); }, x,
-      coefficients.sin_coefficients, 0.0);
+      [k_times_x](double k_times_z, int n) { return sin((n + 1) * k_times_z); },
```

Comment:
Clang-tidy is unhappy here, but I think you can fix that by not capturing `k_times_x` or not having `k_times_z` as an argument to `evaluate_with_function_basis`

Reply:
yeah capturing is no longer necessary